modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/introspection/validation/NameClashModelValidatorTestCase.java

```diff
@@ -91,118 +107,193 @@ public void validModel() {
     validate();
   }
 
-  @Test(expected = IllegalModelDefinitionException.class)
+  @Test
   public void operationClashingWithConfig() {
+    exception.expect(IllegalModelDefinitionException.class);
```

Comment:
instead of putting this line at the beginning of every method, initialise the rule to this value and only set it to `none()` when you expect the validation to pass